modules/nbde-unexpected-loss-of-network-connectivity.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// security/nbde-implementation-guide.adoc
+
+[id="nbde-unexpected-loss-of-network-connectivity_{context}"]
+= Unexpected loss of network connectivity
+
+If the network disruption is unexpected and a server reboots, consider the following scenarios:
+
+* If any servers are still online, ensure that they do not reboot until network connectivity is restored. This is not applicable for single node OpenShift (SNO) clusters, where there is only one server.
```

Comment:
Good catch!  We've been trying to keep the language consistent (where "server" always means "the tang server" and "node" means "The Openshift SNO node"), and this is a departure from that otherwise consistent language.

First paragraph should should be "and a node reboots".

First bullet should be "If any nodes are still online", and maybe we should change the 2nd sentence since it seems odd to end it with "where these is only one node" (which would be better for consistency) because we already said it's "single node"?

Next bullet should also be changed to "node will remain offline"

Reply:
Made all changes. Deleted "where these is only one node".